stemming/stemming.py

```diff
@@ -102,16 +102,18 @@ def prepare_stem_mapping(self, text_to_stem_list):
         return words
 
     def stemmify(self, table):
-        steamed_table = copy.deepcopy(table)
-        only_text_respective = [row[0] for row in steamed_table]
+        stemmed_table = copy.deepcopy(table)
+        only_text_respective = [row[0] for row in stemmed_table]
         self.prepare_stem_mapping(only_text_respective)
 
-        for row_i in range(len(steamed_table)):
-            steamed_table[row_i][0] = self.convert_all_text_to_stem_only(only_text_respective[row_i])
-        return steamed_table
+        for row_i in range(len(stemmed_table)):
+            stemmed_table[row_i][0] = self.convert_all_text_to_stem_only(only_text_respective[row_i])
+        return stemmed_table
 
     def convert_all_text_to_stem_only(self, text):
-        return [''.join(from_word_to_stem_cache.get(word, '')) for word in text.split()]
```

Comment:
notice this change is only for english stemming, heb remained the same above row 75